src/features/settings/components/BookmarkSettings/BookmarkSettings.tsx

```diff
@@ -60,17 +61,84 @@ const BookmarkItem = ({ item, appendRef = false }: BookmarkItemProps) => {
 }
 
 export const BookmarkSettings = () => {
-  const { userBookmarks } = useBookmarks()
+  const inputFile = useRef<HTMLInputElement | null>(null)
+  const {initState, userBookmarks } = useBookmarks()
+
+  const ImportBookmarks = () => {
+    inputFile.current?.click()
+  }
+
+  const ExportBookmarks = () => {
+    const blob = new Blob([JSON.stringify(userBookmarks, null, 2)], {
+      type: 'application/json',
+    })
+    const downloadURL = URL.createObjectURL(blog);
```

Comment:
the variable `blog` should be `blob`

Reply:
WTF :laughing: 
Why didn't I got any error? Maybe it was changed by mistake and I didn't tested after that.. :sweat_smile: